website/docs/d/tg_route_reports.html.markdown

```diff
@@ -0,0 +1,56 @@
+---
+
+subcategory: "Transit Gateway"
+layout: "ibm"
+page_title: "IBM : tg_gateways"
+description: |-
+  Manages IBM Cloud Infrastructure Transit Gateways.
+---
+
+# ibm_tg_route_reports
+Retrieve information of an existing IBM Cloud infrastructure transit gateway route reports as a read only data source. For more information about Transit Gateway Route Reports, see [generating and viewing a route report](https://cloud.ibm.com/docs/transit-gateway?topic=transit-gateway-route-reports&interface=ui#generate-route-report-ui).
+## Example usage
```

Comment:
Add one empty line above # Example usage

Reply:
Fixed this